precompiles/avs/avs_test.go

```diff
@@ -193,7 +197,14 @@ func (suite *AVSManagerPrecompileSuite) TestRegisterAVS() {
 
 			// Run precompiled contract
 			bz, err := suite.precompile.Run(evm, contract, tc.readOnly)
-
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
+			suite.CommitAfter(time.Hour*24 + time.Nanosecond)
```

Comment:
Same comment - and instead of using a hardcoded number, please use a common constant which will be used during AVS registration as well as deregistration.